web/html/src/manager/admin/password-policy/password-policy.tsx

```diff
@@ -0,0 +1,208 @@
+import { hot } from "react-hot-loader/root";
+
+import withPageWrapper from "components/general/with-page-wrapper";
+import { Form, Text, Check } from "components/input";
+import { Panel } from "components/panels/Panel";
+import { TopPanel } from "components/panels/TopPanel";
+import { AsyncButton } from "components/buttons";
+import { showErrorToastr, showSuccessToastr } from "components/toastr/toastr";
+
+// Assume you have an API hook similar to useLifecyclePaygActionsApi
+import usePasswordPolicyApi from "manager/admin/password-policy/use-password-policy-api";
```

Comment:
This file does not seem to exist in this pull request, did you perhaps forget to `git add` it?